src/openalea/strawberry/analysis.py

```diff
@@ -10,6 +10,15 @@
 from openalea.mtg.algo import orders
 from openalea.mtg import stat, algo, traversal
 
+from pandas.core.groupby.groupby import DataError
```

Comment:
Put first import of standard Python libraries.
Then 3rd party libs.
Then openalea ones (mtg & co)